config.toml

```
@@ -1,29 +1,106 @@
+# The global section has parameters that apply globally to the relayer operation.
+# https://hermes.informal.systems/config.html
 [global]
-# Valid strategies:
-#   'packets': restricts relaying to packets only, or
-#   'all': relay from all types of events
+
+# Specify the strategy to be used by the relayer. Default: 'packets'
+# Two options are currently supported:
+#   - 'all': Relay packets and perform channel and connection handshakes.
+#   - 'packets': Relay packets only.
 strategy = 'packets'
+
+# Specify the verbosity for the relayer logging output. Default: 'info'
+# Valid options are 'error', 'warn', 'info', 'debug', 'trace'.
 log_level = 'info'
 
+
+# The telemetry section defines parameters for Hermes' built-in telemetry capabilities.
+# https://hermes.informal.systems/telemetry.html
 [telemetry]
+
+# Whether or not to enable the telemetry service. Default: false
 enabled = true
+
+# Specify the IPv4/6 host over which the built-in HTTP server will serve the metrics
+# gathered by the telemetry service. Default: 127.0.0.1
 host = '127.0.0.1'
+
+# Specify the port over which the built-in HTTP server will serve the metrics gathered
+# by the telemetry service. Default: 3001
 port = 3001
 
+# A chains section includes parameters related to a chain and the full node to which
+# the relayer can send transactions and queries.
 [[chains]]
+
+# Specify the chain ID. Required
 id = 'ibc-0'
+
+# Specify the RPC address and port where the chain RPC server listens on. Required
 rpc_addr = 'http://127.0.0.1:26657'
+
+# Specify the GRPC address and port where the chain GRPC server listens on. Required
 grpc_addr = 'http://127.0.0.1:9090'
+
+# Specify the WebSocket address and port where the chain WebSocket server
+# listens on. Required
 websocket_addr = 'ws://127.0.0.1:26657/websocket'
+
+# Specify the maximum amount of time (duration) that the RPC requests should
+# take before timing out. Default: 10s (10 seconds)
 rpc_timeout = '10s'
+
+# Specify the prefix used by the chain. Required
 account_prefix = 'cosmos'
+
+# Specify the name of the private key to use for signing transactions. Required
+# See the Adding Keys chapter for for more information about managing signing keys:
+#   https://hermes.informal.systems/commands/keys/index.html#adding-keys
 key_name = 'testkey'
+
+# Specify the store prefix used by the on-chain IBC modules. Required
 store_prefix = 'ibc'
+
+# Specify the maximum amount of gas to be used as the gas limit for a transaction.
+# Default: 300000
 max_gas = 3000000
-gas_price = { price = 0.001, denom = 'stake' }
+
+# Specify the price per gas used of the fee to submit a transaction and
+# the denomination of the fee. Required
+gas_price = {
+  price = 0.001,
+  denom = 'stake'
+}
```

I think adding newline here will not parse correctly.

Specifically, this is the error:

```
> hermes -c config.toml query clients ibc-0
error: hermes fatal error: parse error: expected a table key, found a newline at line 69 column 14
```

```suggestion
gas_price = { price = 0.001, denom = 'stake' }
```

---

Interesting. Didn't know toml has such a quirk. Fixed.